packages/avatars/stories/examples/ChromeUsage.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { Chrome, Body, Header, HeaderItem, HeaderItemIcon } from '@zendeskgarden/react-chrome';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import GridIcon from '@zendeskgarden/svg-icons/src/16/grid-2x2-stroke.svg';
+
+export const ChromeUsage: Story = () => {
+  const CustomHeaderItem = HeaderItem as any;
+
+  return (
+    <Chrome isFluid style={{ height: 'auto' }}>
+      <Body>
+        <Header>
+          <CustomHeaderItem as="button">
```

Comment:
It looks like there's an a11y violation shown in the addon-a11y tab. Since this element is rendered as a button, could it have an `aria-label`? I think that might address the violation.

Reply:
Updated.